coalib/processes/Processing.py

```diff
@@ -466,6 +468,20 @@ def yield_ignore_ranges(file_dict):
                                filename,
                                line_number, 1,
                                end_line, len(file[end_line - 1])))
+                elif 'noqa ' in line:
+                    end_line = min(line_number + 1, len(file))
+                    yield (get_ignore_scope(line, 'noqa '),
+                           SourceRange.from_values(
+                               filename,
+                               line_number, 1,
+                               end_line, len(file[end_line - 1])))
+                elif 'noqa' in line:
```

Comment:
The second one doesn't have a space after the `noqa`, I needed this to call the 'all bears', otherwise `noqa` would be looking for a string after the keyword and we want `noqa` to do 2 things: 
1)If no bears are specified, `noqa` will run for all bears.
2)If a bear is specified, `noqa` will run for that bear only.